py3status/modules/dpms.py

```diff
@@ -48,30 +52,43 @@ class Meta:
             ],
         }
 
+    def post_config_hook(self):
+        self.color_on = self.py3.COLOR_ON or self.py3.COLOR_GOOD
+        self.color_off = self.py3.COLOR_OFF or self.py3.COLOR_BAD
+
     def dpms(self):
         """
         Display a colorful state of DPMS.
         """
-        self.run = system('xset -q | grep -iq "DPMS is enabled"') == 0
-        icon = self.icon_on if self.run else self.icon_off
+        run = 'DPMS is Enabled' in self.py3.command_output('xset -q')
+
+        if run:
```

Comment:
Python is crazy. Yes. I like it. Less lines. How to 1s delay `xset dpms force off`? Time to get away a bit.. Avoids waking up DPMS again.